README.md

```diff
@@ -4,41 +4,42 @@
+Wayfire is a wayland compositor based on wlroots. It aims to create a customizable, extendable and lightweight environment without sacrificing its appearance. If you want to gain a better impression at what it can do, see the demo videos on youtube: [link](https://www.youtube.com/playlist?list=PLb7YRKEhWEBUIoT-a29UoJW9mhfzjpNle)
 
 # Build and install
 
-To build wayfire, you'll need `cmake`, `glm`, `gio-2.0`, `freetype2`, `cairo` and `wlroots` and its build dependencies. Optional dependencies are `gdk-2.0`/`gtk2` and `gdk-pixbuf-2.0`(for loading jpeg backgrounds). When ready, simply clone this repo, compile and install:
+To build wayfire, you'll need `glm`, `wf-config` and`wlroots` + its build dependencies. When ready, simply clone this repo, compile and install:
 
 ```
-git clone https://github.com/ammen99/wayfire && cd wayfire
-meson build --prefix=/usr --buildtype=release 
+git clone https://github.com/WayfireWM/wayfire && cd wayfire
+meson build --prefix=/usr --buildtype=release
 ninja -C build && sudo ninja -C build install
 ```
 
 **Before running Wayfire, copy the default configuration file which is located in the root of the repository and place it in:**
 ```
 cp wayfire.ini.default ~/.config/wayfire.ini
 ```
+It is also advisable to install https://github.com/WayfireWM/wf-shell in order to get a background and a panel. Just follow the instructions in the README of wf-shell.
 
-You can adjust background, panel properties (font family/size, which launchers to use, etc.) and key/button bindings in this file to your liking. To start wayfire, just execute `wayfire` from a TTY. If you encounter any issues, please read [debug report guidelines](https://github.com/ammen99/wayfire/wiki/Debugging-problems) and open a bug in this repo. You can also write in gitter.
+To start wayfire, just execute `wayfire` from a TTY. If you encounter any issues, please read [debug report guidelines](https://github.com/ammen99/wayfire/wiki/Debugging-problems) and open a bug in this repo. Or you can also write in gitter.
 # Project status
 
-**IMPORTANT**: Although many of the features one can expect from a WM are implemented, Wayfire should be considered as **WIP** and **pre-alpha**. In my setup it works flawlessly, but this project hasn't been thoroughly tested, so there are a lot of bugs to be expected and to be fixed.
+**IMPORTANT**: Although many of the features one can expect from a WM are implemented, Wayfire should be considered as **(pre-)alpha** quality. In my setup it works just fine, but the project hasn't been extensively tested, so there are a lot of bugs to be expected and to be fixed. Bug reports are welcome!
 
 Currently supported:
 1. Seamless integration of both native wayland & Xwayland clients
 2. Workspaces (or more like viewports if you are familiar with compiz)
 3. Configurable bindings
```

Comment:
Perhaps expand to bring out multiple bindings can be configured for each action?

Reply:
This also includes things like Move's activate, where you can bind only a single binding